modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/trait/WithEventBus.java

```diff
@@ -39,12 +43,31 @@ default void publish(String address, Object message, DeliveryOptions options) {
 	}
 
 	/**
-	 * @see io.vertx.core.eventbus.EventBus#publish(String, Object, DeliveryOptions)
+	 * @param information additional information that should be appended to the message headers
+	 * @see io.vertx.core.eventbus.EventBus#publish(String, Object)
+	 */
+	default void publish(String address, Object message, Information... information) {
+		var headers = InformationUtils.allToHeaders(information);
+		publish(address, message, new DeliveryOptions().setHeaders(headers));
```

Comment:
What is the difference between our information and the headers here? Why all the trouble. I don't see the big picture. Maybe it would be nice to write in the issue the problem with the vertX approach of adding header information to the message.